packages/devtools_app/lib/src/screens/deep_link_validation/deep_link_list_view.dart

```diff
@@ -72,112 +76,69 @@ class _DeepLinkListViewMainPanel extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final controller = Provider.of<DeepLinksController>(context);
-    return ValueListenableBuilder<List<LinkData>?>(
-      valueListenable: controller.linkDatasNotifier,
-      builder: (context, linkDatas, _) {
-        if (linkDatas == null) {
-          return const CenteredCircularProgressIndicator();
-        }
-        return Column(
-          children: [
-            AreaPaneHeader(
-              title: Text(
-                'All deep links',
-                style: Theme.of(context).textTheme.bodyLarge,
+    final textTheme = Theme.of(context).textTheme;
+
+    return ValueListenableBuilder<DisplayOptions>(
```

Comment:
I tried MultiValueListenableBuilder but the widget is not updated instantly so I am still using stacked ValueListenableBuilder

Reply:
We use `MultiValueListenableBuilder` all over DevTools, so I would be surprised if there was a bug in its lifecycle that we haven't encountered somewhere else already. Perhaps an issue of when `.value =` is being called on the notifiers you are passing to the `MultiValueListenableBuilder` widget? Can you add a TODO here to move this to `MultiValueListenableBuilder` so that the tech debt is at least tracked by a TODO? thanks